### PR TITLE
[CP-831] Move and repair test_device_diagnostics.py

### PR DESCRIPTION
**Pull request by niepiekm**
Moved test script out of pytest/service-desktop
Cleaned up and fixed test code for multiple log/crash dump files

### CODE REVIEW

File: test/pytest/test_device_diagnostics.py [hunk 1]
```diff
@@ -16,12 +16,11 @@ def test_device_list_logs(harness):
...
 
+    log.info("Available log files:")
     for file in ret.files:
-        print(file)
-        assert True
-        return
+        log.info(file)
 
-    assert False
+    assert True
```
Comment (pawel-mudita) [inline test/pytest/test_device_diagnostics.py:23]: Do we need this assertion?

File: test/pytest/test_device_diagnostics.py [hunk 2]
```diff
@@ -34,12 +33,11 @@ def test_device_list_crash_dumps(harness):
...
 
+    log.info("Available crash dump files:")
     for file in ret.files:
-        print(file)
-        assert True
-        return
+        log.info(file)
 
-    assert False
+    assert True
```
Comment (pawel-mudita) [inline test/pytest/test_device_diagnostics.py:40]: Same here

File: test/pytest/test_device_diagnostics.py (hunk 1 above)
Comment (niepiekm) [reply test/pytest/test_device_diagnostics.py]: Actually, it can be removed

**Review by pawel-mudita — APPROVED** [review]

File: test/pytest/test_device_diagnostics.py [hunk 3]
```diff
@@ -12,16 +12,12 @@ def test_device_list_logs(harness):
     ret = GetDiagnosticFilesList(DiagnosticsFileList.LOGS).run(harness)
 
     if not ret.files:
-        log.info("No log files")
-        assert True
+        log.warning("No log files")
```
Comment (pholat) [inline test/pytest/test_device_diagnostics.py:15]: there was assert here, maybe raise an exception? similarly below.

**Review by pholat — APPROVED** [review]

**Review by SP2FET — APPROVED** [review]

**Pull request merged**